README.md

```diff
@@ -257,6 +332,24 @@ End with agreements:
 
 One of the quickest ways to derail honest communication is by presuming to know the intentions of the other person. Remember to "stay on your side of the net"—focus on specific, observable behavior and ask the other person about their motives and intentions.
 
+## Being a point person :point_up:
+
+Point people are our guides to help us define our roles and set and meet goals on an ongoing basis. Everyone in the company has a point person—even the CEO. Point people are not managers, and that means they people have a unique set of responsibilities at Able Health. These include:
+
+* Working with team members to define roles and set goals in January and July
+
+* Meeting one-on-one with team members on a regular basis
+
+* Supporting team members in achieving their goals
+
+* Working with team members to reflect on and update roles
+
+* Helping team members navigate conflicts related to roles and goals
+
+Point people _do not_ have the inherent authority to assign work or make decisions related to compensation, promotion, demotion, or firing.
```

Comment:
Who has the authority for promotions/demotion/firing and how does that work? E.g. if a junior engineer meets agreed upon requirements for a promotion, how is that managed?